### PR TITLE
Rename tokens

### LINKED ISSUE #14: Uniswap connects to the wrong chain if MM is not connected to Hemi Sepolia
If MM is connected to another chain (Say "Ethereum Sepolia"), when connecting, it shows tokens from that chain, instead of switching to Hemi Sepolia (or showing something like "Wrong chain")

https://github.com/hemilabs/interface/assets/352474/18573432-0a54-46b0-bd2d-17b15310a183

### PR DESCRIPTION
**Pull request by leanazulyoro**
In this PR:

- Removes many references to unsupported chains
- Renames tokens from thDAI->DAI and thETH->ETH
- Makes HEMI_SEPOLIA the default chain. It will now prompt to "Connect to HEMI_SEPOLIA" if connected to any other unsupported chain, and won't show any token from those chains.
- Fixes an issue with allowance that prevented swapping from ERC-20 tokens (eg.: DAI->ETH)

## Screenshots
"Connect to Hemi Sepolia":

https://github.com/hemilabs/interface/assets/1864435/e0f35bc9-17ea-46d8-b843-14d9112816d0

A successful swap from DAI->ETH:

https://github.com/hemilabs/interface/assets/1864435/cb26db03-0486-40a3-90c1-f7f65c9ea305

Fixes #14

### CODE REVIEW

**Review by gabmontes — APPROVED** [review]
uACK

File: apps/web/src/constants/tokens.ts [hunk 1]
```diff
@@ -522,6 +554,9 @@ export const TOKEN_SHORTHANDS: { [shorthand: string]: { [chainId in ChainId]?: s
     [ChainId.SEPOLIA]: USDC_SEPOLIA.address,
     [ChainId.AVALANCHE]: USDC_AVALANCHE.address,
   },
+  USDT: {
```
Comment (gndelia) [inline apps/web/src/constants/tokens.ts:557]: should we add USDC a few lines above?

File: apps/web/src/hooks/useAssetLogoSource.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export function getInitialUrl(
   const networkName = chainId ? chainIdToNetworkName(chainId) : 'ethereum'
   const checksummedAddress = isAddress(address)
 
-  if (chainId && isCelo(chainId) && address === nativeOnChain(chainId).wrapped.address) {
+  if (chainId && address === nativeOnChain(chainId).wrapped.address) {
     return celoLogo
```
Comment (gndelia) [inline apps/web/src/hooks/useAssetLogoSource.ts:53]: Just to confirm, is this correct? Has `celoLogo` been replaced with ours? or would this never execute?

File: apps/web/src/hooks/useStablecoinPrice.ts [hunk 3]
```diff
@@ -41,7 +41,7 @@ export const STABLECOIN_AMOUNT_OUT: { [key in SupportedInterfaceChain]: Currency
   [ChainId.ARBITRUM_GOERLI]: CurrencyAmount.fromRawAmount(USDC_ARBITRUM_GOERLI, 10_000e6),
   [ChainId.POLYGON_MUMBAI]: CurrencyAmount.fromRawAmount(USDC_POLYGON_MUMBAI, 10_000e6),
   [ChainId.CELO_ALFAJORES]: CurrencyAmount.fromRawAmount(CUSD_CELO_ALFAJORES, 10_000e6),
-  [ChainId.HEMI_SEPOLIA]: CurrencyAmount.fromRawAmount(TESNET_DAI_HEMI_SEPOLIA, 10_000e18),
+  [ChainId.HEMI_SEPOLIA]: CurrencyAmount.fromRawAmount(DAI_HEMI_SEPOLIA, 10_000e18),
```
Comment (gndelia) [inline apps/web/src/hooks/useStablecoinPrice.ts:44]: I think this should use USDC, but I see you're not adding it as part of this PR so... just commenting it for future reference 😄

**Review by gndelia — APPROVED** [review]
LGTM, left some comments but may not be blockers so... resolve and merge them if you think they are not correct

File: apps/web/src/hooks/useStablecoinPrice.ts (hunk 3 above)
Comment (leanazulyoro) [reply apps/web/src/hooks/useStablecoinPrice.ts:44]: Not sure if it should use USDC, some chains use USDC, others USDT or DAI.

Comment (gndelia) [reply apps/web/src/hooks/useStablecoinPrice.ts:44]: well, optimism uses dai, so let's use that 🤷🏽

File: apps/web/src/hooks/useAssetLogoSource.ts (hunk 2 above)
Comment (leanazulyoro) [reply apps/web/src/hooks/useAssetLogoSource.ts:53]: I'll be removing this whole conditional in an upcoming PR, but I'll keep this one clean and restore this, it just slipped, thanks.

File: apps/web/src/constants/tokens.ts (hunk 1 above)
Comment (leanazulyoro) [reply apps/web/src/constants/tokens.ts:557]: I'll have this clean and add USDC in an upcoming PR

File: apps/web/src/hooks/useAssetLogoSource.ts (hunk 2 above)
Comment (leanazulyoro) [reply apps/web/src/hooks/useAssetLogoSource.ts:53]: I'll just do it in next PR 😅

**Pull request merged**